src/python/ddapp/roboturdf.py

```diff
@@ -203,6 +204,11 @@ def loadRobotModel(name, view=None, parent='planning', urdfFile=None, color=None
     obj.setProperty('Visible', visible)
     obj.setProperty('Name', name)
     obj.setProperty('Color', color or getRobotGrayColor())
+    if colorMode == 'Texture': # fix for #111
+        obj.setProperty('Textures', True)
+    elif colorMode == 'URDF Colors':
```

Comment:
since useUrdfColors is already set to True in the RobotModelItem constructor, shouldn't this be handling the Solid Color case? Set useUrdfColors to false and call obj._updateModelColor()